public/javascripts/ng-controllers.js

```diff
@@ -97,6 +97,7 @@ springInitiative.controller('mainController', function($scope, $http) {
       })
   };
 
+  // TODO: .success and .error are deprecated
```

Comment:
@BrennaManning you should use the `.then` function. implementation is very similar. check these [docs](https://docs.angularjs.org/api/ng/service/$http)